packages/babel-types/test/builders/es2015/templateElement.js

```diff
@@ -21,5 +21,31 @@ describe("builders", function() {
         expect(() => t.templateElement("foo")).toThrowErrorMatchingSnapshot();
       });
     });
+    describe("templateLiteral", function() {
+      it("should validate", function() {
+        const foo = t.templateElement({ raw: "foo" });
+        const bar = t.templateElement({ raw: "bar" });
+
+        expect(() => t.templateLiteral([foo], [])).toMatchSnapshot();
```

Comment:
Don't wrap this in an arrow function. That's why we're getting `[Function]` in the snapshots.